pkg/authorization/decision.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package authorization
+
+import "k8s.io/apiserver/pkg/authorization/authorizer"
+
+const (
+	DecisionNoOpinion = "noopinion"
```

Comment:
Can we revert these back to NoOpinion, Allowed, Denied?

Reply:
sure but this was to make it somewhat aligned to upstream reasons in audit. I have no strong opinions here.